config/cfp-data.json

```diff
@@ -0,0 +1,7 @@
+{
+    "EventStartDate": "30th October 2024",
+    "EventEndDate": "30th October 2024",
+    "CallStartDate": "5th August 2024",
+    "CallEndDate": "30th August 2024",
+    "Edition": "Online Edition"
+}
```

Comment:
Why do we have this in a separate file?

Reply:
So that changing the data becomes easier.
I thougth this would keep things aligned and lead to less errors as contributors wont touch this file hence not edit these details.

Reply:
Don't we have a similar file for other events?

Reply:
Yeah we do but the data in them dont fit with requirements here :thinking: